framework/test/mocha/integration/process_transactions.js

```diff
@@ -346,6 +347,16 @@ describe('processTransactions', () => {
 						);
 					});
 				});
+
+				it('should call votes.apply with correct params', async () => {
```

Comment:
Tests which spy or stub should be unit tests. No?

Reply:
Here we can test that stateStore value is updated as expected if we want this to be an integration test

Reply:
I added this test here because the unit test for this method was skipped as reported in the issue https://github.com/LiskHQ/lisk-sdk/issues/3987

Reply:
I think spy is fine for integration test